python/lsst/cp/verify/verifyCrosstalk.py

```diff
@@ -126,3 +130,73 @@ def verify(self, calib, statisticsDict, camera=None):
             success = False
 
         return verifyStats, success
+
+    def repackStats(self, statisticsDict, dimensions):
+        """Repack information into flat tables.
+
+        This method should be redefined in subclasses.
```

Comment:
If this is a subclass redefinition, it doesn't actually need a docstring.  You can add a comment `# docstring inherited` to make that clear.